### PR TITLE
Update WAN Rest API with target scope information [HZ-4594]

### PR DESCRIPTION
**Pull request by JamesHazelcast**
Some WAN API endpoints apply to the entire cluster, while some only apply to the receiving member. This commit adds notes to each endpoint specifying what the scope of its impact is.

I'm not sure if the `IMPORTANT:` boxes are the best way to present this, so please feel free to suggest better approaches!

Related to https://hazelcast.atlassian.net/browse/HZ-4594

### CODE REVIEW

File: docs/modules/wan/pages/rest-api.adoc [hunk 1]
```diff
@@ -233,6 +244,8 @@ The following is an example output:
 
 The other stages are `STARTED`, `IN_PROGRESS` and `FAILED`.
 
+IMPORTANT: As WAN synchronization is an operation that occurs across the cluster, this response will represent the WAN sync progress for the entire cluster.
```
Comment (rebekah-lawrence) [inline docs/modules/wan/pages/rest-api.adoc:247]:
```suggestion
IMPORTANT: As WAN synchronization is an operation that occurs across the cluster, this response represents the WAN sync progress for the entire cluster.
```

File: docs/modules/wan/pages/rest-api.adoc [hunk 2]
```diff
@@ -250,4 +263,6 @@ curl -X POST -d "{clusterOnSource}&{clusterPassword}&{wanConfig}" --URL http://1
 ```
 
 The `wanConfig` parameter should be the full configuration as a JSON string.
-See xref:advanced-features.adoc#dynamically-adding-wan-publishers[here] for configuration examples.
\ No newline at end of file
+See xref:advanced-features.adoc#dynamically-adding-wan-publishers[here] for configuration examples.
+
+IMPORTANT: This endpoint will deploy the provided config across all members of the cluster, following the same principles as other dynamic configuration mechanisms.
```
Comment (rebekah-lawrence) [inline docs/modules/wan/pages/rest-api.adoc:268]:
```suggestion
IMPORTANT: This endpoint deploys the provided config across all members of the cluster, following the same principles as other dynamic configuration mechanisms.
```

**Review by rebekah-lawrence — APPROVED** [review]
I've changed to present tense in my suggestions, that's all.

On the IMPORTANT admonition; I would generally use NOTE, but this is used consistently here, and we have yet to complete the style guidelines on the use of each type of admonition, so I think we can leave it as you have it. 

Approved in advance. Thanks James

**Review by Serdaro — APPROVED** [review]
LGTM

**Pull request merged**